# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/power/moz.build
# Commit: f50ff29a1a64
# Full Hash: f50ff29a1a649e3aad723b592bdd2c3c325b3d99
# Author: Vincent Hilla <vhilla@mozilla.com>
# Date: 2023-12-06 05:18:37
# Regressor Bug: 1589554
# File Overlap Count: 1
# Description:
#   Bug 1589554 - Part 1: Screen Wake Lock boilerplate. r=dom-core,smaug,emilio,edgar
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D189508
# ==============================================================================

diff -r 00afe2e46b1b -r f50ff29a1a64 dom/power/moz.build
--- a/dom/power/moz.build	Tue Dec 05 23:29:16 2023 +0000
+++ b/dom/power/moz.build	Tue Dec 05 23:58:06 2023 +0000
@@ -19,9 +19,7 @@
     "components.conf",
 ]
 
-EXPORTS.mozilla.dom += [
-    "WakeLock.h",
-]
+EXPORTS.mozilla.dom += ["WakeLock.h", "WakeLockJS.h", "WakeLockSentinel.h"]
 
 EXPORTS.mozilla.dom.power += [
     "PowerManagerService.h",
@@ -30,6 +28,8 @@
 UNIFIED_SOURCES += [
     "PowerManagerService.cpp",
     "WakeLock.cpp",
+    "WakeLockJS.cpp",
+    "WakeLockSentinel.cpp",
 ]
 
 include("/ipc/chromium/chromium-config.mozbuild")